spec/system/documents/edit_documents_spec.rb

```diff
@@ -56,6 +56,30 @@
       expect(page).to have_content("Public: Yes")
     end
 
+    it "allows for the date received to be edited" do
+      click_link "Edit"
+
+      expect(page).to have_content("Date received")
+
+      fill_in "Day", with: "19"
+      fill_in "Month", with: "11"
+      fill_in "Year", with: "2021"
+
+      click_button("Save")
+      expect(page).to have_content("Date received: 19 November 2021")
+    end
+
+    it "does not allow for a future date to be inserted in created_date" do
+      click_link "Edit"
+
+      fill_in "Day", with: "19"
+      fill_in "Month", with: "11"
+      fill_in "Year", with: "3021"
+
+      click_button("Save")
+      expect(page).to have_content("Date must be today or earlier. You cannot insert a future date.")
+    end
```

Comment:
is there any chance you can port these to Cucumber specs rather than adding to these?

Reply:
I recall you shied away from doing the document specs in cucumber because of having problems loading the pdfs into it?

Reply:
OK. Would you mind if I give it a shot and open a PR to merge into your PR?

Reply:
yes, happy to pair on it as well - was just a bit afraid since I remember it was difficult for you so I figured I wouldn't even know where to start!